cmake/modules/RootBuildOptions.cmake

```diff
@@ -110,6 +110,7 @@ ROOT_BUILD_OPTION(fitsio ON "Read images and data from FITS files, requires cfit
 ROOT_BUILD_OPTION(fortran ON "Enable the Fortran components of ROOT")
 set(gcctoolchain "" CACHE PATH "Path for the gcctoolchain in case not the system gcc is used to build clang/LLVM")
 ROOT_BUILD_OPTION(gdml OFF "GDML writer and reader")
+ROOT_BUILD_OPTION(geo ON "Enable the TGeo package")
```

Comment:
This doesn't make it very clear that disabling geo will disable a big bucket of modules.

Reply:
Right, it would make sense to drop a status message mentioning those